### PR TITLE
format and tidy every file

### PR DESCRIPTION
**Pull request by hcweigand10**
Notes:

Could not easily workaround a "no-nested-ternary" error in input/Typeahead.js (line 46) so I disabled the rule for that specific line.  Someone said on Stackoverflow that it's not the most important rule but idk.

Had to change a number of divs/spans that had click handlers into buttons. I don't think this affected much, but be on the lookout. There's some weirdness with the add customer modal but that mightve already been that way.

the getEmoji util function was using "i" as an iterator in both a nested and outer for loop. Surprised this didn't cause any issues before. Changed inner loop to "j", not sure if this will affect anything, emojis seemed fine before and still okay now.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,19 @@
-import { useState } from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import { useQuery } from "react-query";
+import { useState } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useQuery } from 'react-query';
 
 import {
```
Comment (coleloui) [inline src/App.js:5]: this page is missing some formatting...there are ```;``` and the import from pages should be inline which is interesting.

**Review by coleloui — COMMENTED** [review]
Please reach out if you have any questions id be happy to hop on a google meet call or something to go over the comments!

File: src/App.js [hunk 2]
```diff
@@ -28,21 +25,21 @@ function App() {
...
       if (token) {
-        setUserId(UserId);
-        localStorage.setItem("token", token);
+        setUserId(userId);
+        localStorage.setItem('token', token);
         return;
       }
 
       if (!isLoginPage) {
-        location.replace("/login");
```
[the remark this answers is not in the document]
Comment (hcweigand10) [reply src/App.js:45]: Looks like useNavigate can only be used inside the router, and the router is set up in this component. Probably a workaround, but I'll leave this be for now

File: src/pages/ShiftReport.js [hunk 3]
```diff
@@ -1,80 +1,72 @@
...
+        setDate('No shift with this ID')
       }
     },
-  });
+  })
 
   const getItemCounts = (orders) => {
-    const counts = {};
+    const counts = {}
     setOrders(
```
[the remark this answers is not in the document]
Comment (hcweigand10) [reply src/pages/ShiftReport.js:37]: Love it

**Review by coleloui — APPROVED** [review]
looks good!

**Pull request merged**